src/app/addmarket/page.tsx

```diff
@@ -36,8 +36,14 @@ const AddApiKeyForm: React.FC = () => {
     queryMarketList();
   }, []);
 
+  // Use get method to search all the marketlist
   const queryMarketList = async () => {
-    // Need to be completed.
+    try {
+      const response = await axios.get('/api/keys');
+      setExchangeList(response.data.list);
```

Comment:
Did you forget to check the data before set the list? (Accroding to old version)